lib/phlex/html.rb

```diff
@@ -187,7 +187,8 @@ def text(content)
 				case content
 					when String then content
 					when Symbol then content.name
-					else content.to_s
+					when Integer, Float then content.to_s
```

Comment:
I think it makes sense to have a fast-path for Integer, Float, Symbol and String since these are the most common objects and it would be unusual to want to format them in any other way than this.